src/components/nav/NavBar.jsx

```diff
@@ -55,6 +55,10 @@ const Navbar = () => {
   const selectedSpecialization = useSelector(getSelectedSpecialization);
   const selectedTemplateName = useSelector(getSelectedTemplate);
 
+  const selectedTemplate = data.templates.list
+    .find((elem) => elem.class === profession)
+    ?.builds?.find((elem) => elem.name === selectedTemplateName);
```

Comment:
I was gonna swear that I've done this before! Turns out that I have, in https://github.com/discretize/discretize-gear-optimizer/blob/35f76c0f0bbc16b1686a159e9bf49f6ac198c319/src/assets/presetdata/templateTransform.js#L23-L26

I guess ideally we could extract this somewhere (or go all fancy and make it a selector using redux-toolkit's createSelector), but I don't think it's a big deal. This codebase is not very factored :D